server/mdm/lifecycle/lifecycle.go

```diff
@@ -174,6 +175,7 @@ func (t *HostLifecycle) turnOnDarwin(ctx context.Context, opts HostOptions) erro
 			opts.Platform,
 			tmID,
 			opts.EnrollReference,
+			!opts.HasSetupExperienceItems,
```

Comment:
Meaning "use worker-based device release if the device has no setup experience items to process"

Reply:
This will enqueue the post-DEP enrollment task (which sends commands to install fleetd, bootstrap, account config), and then will force the use of the worker job to release the device when those commands are done, without needing to wait for an orbit/config request.